frontend/src/components/ExamInstanceItem.tsx

```diff
@@ -0,0 +1,112 @@
+// Copyright (C) 2023, BRAIN-LINK UG (haftungsbeschränkt). All Rights Reserved.
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-ScanHub-Commercial
+// ExamItem.tsx is responsible for rendering a single exam item in the exam list of the patient view.
+
+import * as React from 'react'
+import { useMutation } from 'react-query'
+
+// Mui joy components
+import IconButton from '@mui/joy/IconButton'
+import ListItemButton from '@mui/joy/ListItemButton'
+import ListItem from '@mui/joy/ListItem'
+// import ListItemDecorator from '@mui/joy/ListItemDecorator'
+import ListItemContent from '@mui/joy/ListItemContent'
+import Typography from '@mui/joy/Typography'
+import Dropdown from '@mui/joy/Dropdown';
+import MenuButton from '@mui/joy/MenuButton';
+import MenuItem from '@mui/joy/MenuItem';
+import Menu from '@mui/joy/Menu'
+
+// Icons
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz'
+import ListAltIcon from '@mui/icons-material/ListAlt'
+
+// Sub-components, interfaces, client
+// import ExamModal from './ExamModal'
+import LoginContext from '../LoginContext';
+import WorkflowFromTemplateModal from './WorkflowFromTemplateModal'
+import { InstanceInterface } from '../interfaces/components.interface'
+import { ExamOut } from '../generated-client/exam'
+import { examApi } from '../api'
+
+
+// function ExamInstanceItem({ data: exam, refetchParentData, isSelected }: ComponentProps<ExamOut>) {
+function ExamInstanceItem({data: exam, refetchParentData}: InstanceInterface<ExamOut>) {
+
+  const [user, ] = React.useContext(LoginContext);
+  // const [examModalOpen, setExamModalOpen] = React.useState(false)
+  const [modalOpen, setModalOpen] = React.useState(false)
+
+  const deleteExam = useMutation(async () => {
+    await examApi.examDeleteApiV1ExamExamIdDelete(
+      exam.id, {headers: {Authorization: 'Bearer ' + user?.access_token}}
+    ).then(() => {
+      refetchParentData()
+    })
+  })
+
+  // const updateExam = useMutation(async (data: Exam) => {
```

Comment:
debug log

Reply:
?

Reply:
is this really something that should block the merge of the two repos?

Reply:
Actually this is not a debug log. The mutation is required to update an exam instance. This feature will be implemented soon.